addons/web_editor/views/editor.xml

```diff
@@ -7,6 +7,10 @@
 </template>
 
 <template id="wysiwyg" name="Wysiwyg Editor">
```

Comment:
Still would rename this into something like `compiled_assets_wysiwyg`... 'wysiwyg' could be anything.